languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/ignore.txt

```diff
@@ -450,4 +450,10 @@ taboire
 tabarnane
 tigidou
 cheum
-cheums
\ No newline at end of file
+cheums
+ROC
```

Comment:
Don't we have another file for acronyms?

Reply:
precision: depends if you want to use them as words (like, idk, "RATP", "CAF", which need a gender and number usually). If not ignore.txt is a good place.